tools/helper.psm1

```diff
@@ -89,24 +89,21 @@ function New-gRPCAutoGenCode
         [switch] $Force
     )
 
-    if ($Force -or -not (Test-Path "$RepoRoot/src/Messaging/FunctionRpc.cs") -or
-                   -not (Test-Path "$RepoRoot/src/Messaging/FunctionRpcGrpc.cs"))
+    if ($Force -or -not (Test-Path "$RepoRoot/src/Messaging/gRPC/FunctionRpc.cs") -or
+                   -not (Test-Path "$RepoRoot/src/Messaging/gRPC/FunctionRpcGrpc.cs"))
```

Comment:
```suggestion
                   -not (Test-Path "$RepoRoot/src/Messaging/Protobuf/FunctionRpcGrpc.cs"))
```

or something just because gRPC is the transfer layer... protobuf is the specification...